docs/docs/atom-archive/getting-started/sections/installing-atom.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Installing Atom
+---
+### Installing Atom
+
+To get started with Atom, we'll need to get it on your system. This section will go over installing Atom on your system as well as the basics of how to build it from source.
+
+Installing Atom should be fairly simple. Generally, you can go to https://atom.io and you should see a download button as shown here:
+
+{{#mac}}
+
+![Download buttons on https://atom.io](../../images/mac-downloads.png "Download buttons on https://atom.io")
```

Comment:
Using the same text for `title` and `alt` attributes should be avoided, they serve very different purposes. It's annoying for people who rely on screenreaders to have the same text twice. Alt-texts are a delicate matter that requires a lot of thought to get them right.